core/domain/email_manager_test.py

```diff
@@ -88,6 +90,198 @@ def test_sender_id_validation(self):
         # pylint: enable=protected-access
 
 
+class ExplorationMemebershipEmailTests(test_utils.GenericTestBase):
+    """Tests that sending exploration membership email works as expected."""
+
+    def setUp(self):
+        super(ExplorationMemebershipEmailTests, self).setUp()
+
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+
+        self.signup(self.NEW_USER_EMAIL, self.NEW_USER_USERNAME)
+        self.new_user_id = self.get_user_id_from_email(self.NEW_USER_EMAIL)
+
+        self.exploration = self.save_new_default_exploration(
+            'A', self.editor_id, 'Title')
+
+        self.email_subject_tempate = (
```

Comment:
Is that space you left at the beginning of all `'    <li>------</li>'` points intentional ?